dtschema/schemas/bootph.yaml

```diff
@@ -85,4 +85,23 @@ properties:
     description:
       Include this node in all phases (for U-Boot see enum u_boot_phase).
 
+allOf:
+  - if:
+      anyOf:
+        - required: [ bootph-pre-sram ]
+        - required: [ bootph-verify ]
+        - required: [ bootph-pre-ram ]
+        - required: [ bootph-some-ram ]
+        - required: [ bootph-all ]
+    then:
+      description: Parent nodes don't need bootph tags
```

Comment:
They don't need the same tag, but they might need a different one. I think the rule will be that if a child has a particular phase tag then it is implied in all its parents

Reply:
That should be doable. What about a child with bootph-all and parent with something else?

Reply:
In that case we are implying bootph-all in the parent so it doesn't matter what tags are present